CHANGELOG.md

```diff
@@ -36,11 +36,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * Add support for creating, updating, removing, finding, and iterating over `Record`s #54
 * Add migration support for v039 account into v040 attributes module #100
-* Implemented v39 to v40 migration for name module.
```

Comment:
@iramiller This was listed in the `v0.1.4` release but also duplicated in `Unreleased`. Does this have relevance to v0.1.5?

Reply:
I think that is a 0.1.4 thing only...